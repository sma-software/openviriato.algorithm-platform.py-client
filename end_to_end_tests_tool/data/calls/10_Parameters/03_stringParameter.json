{
    "restResource": "/parameters/{key}",
    "urlParameters": [ "stringParameter" ],
    "httpMethod": "GET",
    "requestBody": null,
    "CSharpMethodName": "GetStringAlgorithmParameter(key)",
    "CSharpMethodParameterTypes": [ "System.String" ],
    "subcategory": "Get Parameters"
}
