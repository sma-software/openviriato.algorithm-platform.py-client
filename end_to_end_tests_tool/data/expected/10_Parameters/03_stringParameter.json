"input of string text field"